app.js

```diff
@@ -84,7 +84,7 @@ app.post('/delete-fileqcs', fileQc.deleteFileQcs);
 app.post('/case', caseController.addCaseArchive);
 app.get('/case/:caseIdentifier', caseController.getCaseArchive);
 app.put(
-  `/case/:caseIdentifier/${urls.filesCopiedToOffsiteStagingDir}`,
+  `/case/:caseIdentifier/${urls.filesCopiedToOffsiteStagingDir}/:batchId`,
```

Comment:
Do `/case/1234/copied-to-offsite-staging/12345` and `/case/1234/copied-to-offsite-staging/56789` represent unique resources?

Reply:
No. I'll move batch ID to body parameters.